traits-stubs/traits-stubs/trait_types.pyi

```diff
@@ -612,7 +612,8 @@ class Date(_TraitType[_OptionalDate, _OptionalDate]):
     def __init__(
             self,
             default_value: datetime.date = ...,
-            allow_datetime: bool = False,
+            allow_datetime: bool = ...,
```

Comment:
This is an unrelated drive-by fix. The default for `allow_datetime` is `True`, not `False`, but the stubs don't actually care, so we replace with a `...` so that we don't have to update the stubs in case the default changes.